lib/gosu.rb

```diff
@@ -217,13 +226,13 @@ def self.font_alignment_flags(mode)
     when :justify
       3
     else
-      return mode if mode.is_a?(Numeric)
+      return flags if flags.is_a?(Numeric)
 
-      raise ArgumentError, "No such mode: #{mode}"
+      raise ArgumentError, "No such font alignment: #{flags}"
     end
   end
 
-  def self.blendmode(mode)
+  def self.blend_mode(mode)
```

Comment:
I've pushed this relatively unrelated renaming to trigger a new CI build. I just realized that AlphaMode is a really dumb name in C++ and it should be called BlendMode, and then this method here should have an underscore.